Documentation violates proselint "But" rule.

```
 docs/why.md
 6:62    warning  'implement' is too wordy.       write-good.TooWordy
 7:1     warning  'purchase' is too wordy.        write-good.TooWordy
 7:19    warning  'It is' is too wordy.           write-good.TooWordy
 7:51    warning  'be built' may be passive       write-good.Passive
                  voice. Use active voice if you
                  can.
 44:57   warning  'in addition' is too wordy.     write-good.TooWordy
 48:44   warning  'be written' may be passive     write-good.Passive
                  voice. Use active voice if you
                  can.
 49:43   warning  'in addition' is too wordy.     write-good.TooWordy
 55:39   warning  'be changed' may be passive     write-good.Passive
                  voice. Use active voice if you
                  can.
 66:36   warning  'implement' is too wordy.       write-good.TooWordy
 70:1    error    Do not start a paragraph with   proselint.But
                  a 'but'.
 118:29  warning  'Only' is a weasel word!        write-good.Weasel
 121:48  warning  'it is' is too wordy.           write-good.TooWordy
 123:1   error    Do not start a paragraph with   proselint.But
                  a 'but'.
 123:29  warning  'few' is a weasel word!         write-good.Weasel
 130:5   warning  'exactly' is a weasel word!     write-good.Weasel
 134:24  warning  'exactly' is a weasel word!     write-good.Weasel
 134:38  warning  'were set' may be passive       write-good.Passive
                  voice. Use active voice if you
                  can.
 142:34  warning  'multiple' is too wordy.        write-good.TooWordy
 143:7   warning  'multiple' is too wordy.        write-good.TooWordy
 188:1   error    Do not start a paragraph with   proselint.But
                  a 'but'.
 194:22  warning  'be sent' may be passive        write-good.Passive
                  voice. Use active voice if you
                  can.
 194:31  warning  'exactly' is a weasel word!     write-good.Weasel
 277:15  warning  'a number of' is too wordy.     write-good.TooWordy
 277:27  warning  'really' is a weasel word!      write-good.Weasel
 279:30  warning  'were defined' may be passive   write-good.Passive
                  voice. Use active voice if you
                  can.
 282:26  warning  'really' is a weasel word!      write-good.Weasel
 285:1   error    Do not start a paragraph with   proselint.But
                  a 'but'.
 287:6   warning  'It is' is too wordy.           write-good.TooWordy
```

--- a/docs/why.md
+++ b/docs/why.md
@@ -61,14 +61,15 @@ We have problems on all three layers of the code: `purchase`, `notify_user` and
    `notify_user` function when they are about to add eighth condition.
 2. **Add context variable instead of another argument.** This little `ctx`
    argument instead of email address, phone number, product price and shipment
-   destination will save us a lot of typing between function calls. But we still
-   have bunch of `if` statements.
+   destination will save us a lot of typing between function calls.
+   Unfortunately, we still have bunch of `if` statements.
 3. **Copy-paste this module and re-implement each feature separately.** Each
    feature will be readable and simple on it's own, but you will regret this
    decision when usage of `request_payment` or `calculate_price` changes.
 
-But we still have the same problem. We can't substitute implementation details
-of low level code without changing high level policies. Let's try…
+Unfortunately, we still have the same problem. We can't substitute
+implementation details of low level code without changing high level policies.
+Let's try…
 
 ## Inheritance
 
@@ -120,8 +121,8 @@ has several advantages.
    layer of abstraction in the system. We can add any number of methods or
    attributes is the child classes. Looks like it is a reasonable approach.
 
-But this code hides quite a few problems underneath the false premise of
-understandability and cleanness.
+Unfortunately, this code hides quite a few problems underneath the false premise
+of understandability and cleanness.
 
 1. **God object.** One class contains methods related to every single layer of
    abstraction in the system. It's hard to manage two hundred methods in the
@@ -185,7 +186,7 @@ Someone might say this is an improvement over one huge class.
 2. **Better code reuse.** We can use the same notification mechanism in
    different classes with just one line of code.
 
-But there are a lot of problems too. During a debugging session,
+Unfortunately, there are a lot of problems too. During a debugging session,
 
 1. **In the `get_notification_text` you have no idea who set up
    `payment_details`.**
@@ -282,7 +283,7 @@ This code has a number of really good characteristics.
 2. **Your system becomes really configurable.** You can **inject** pretty much
    any implementation without changing high-level code.
 
-But there one unfortunate consequence of this style
+Unfortunately, there one unfortunate consequence of this style
 
 1. **It is too much boilerplate on the initiation stage.**
 

